src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -39,13 +39,361 @@ namespace std
 }
 */
 
-#define NEW_SHADERS
+namespace
+{
+    constexpr const char* VERT_SHADER_NO_UV = 
+R"(#version 430 core
+
+layout (location = 0) in vec3 vPos;
+layout (location = 3) in vec3 vNormal;
+
+layout (location = 0) out vec3 LocalPos;
+layout (location = 1) out vec3 ViewPos;
+layout (location = 2) out vec3 Normal;
+
+layout (set = 1, binding = 0, row_major, std140) uniform UBO {
+    mat4 MVP;
+    mat4 MV;
+    mat3 NormalMat;
+    vec3 EyePos;
+} CamData;
```

Comment:
This gets split in two, the UBO declaration, and the data struct.

data struct goes to:
- GLSL a common include that's builtin
- a matching C++ `struct` with correct alignment in some aptly named directory and header inside `irr/asset/`

likewise for the UBO declaration

Reply:
wrap this in a `#ifndef + #define` guard as well